<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ICPVTK</class>
 <widget class="QMainWindow" name="ICPVTK">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ICPVTK</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QVTKOpenGLNativeWidget" name="openGLWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>751</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBasic">
    <property name="title">
     <string>BasicShape</string>
    </property>
    <addaction name="actionCubic"/>
    <addaction name="actionSphere"/>
    <addaction name="actionCylinder"/>
    <addaction name="actionCone"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuBasic"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSphere">
   <property name="text">
    <string>Sphere</string>
   </property>
  </action>
  <action name="actionCone">
   <property name="text">
    <string>Cone</string>
   </property>
  </action>
  <action name="actionCylinder">
   <property name="text">
    <string>Cylinder</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionCubic">
   <property name="text">
    <string>Cubic</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKOpenGLNativeWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>QVTKOpenGLNativeWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
